src/Core/Commander/Providers/BuildingBox_Provider.js

```diff
@@ -70,8 +70,7 @@ BuildingBox_Provider.prototype.getData = function getData(bbox, altitude) {
 };
 
 BuildingBox_Provider.prototype.generateMesh = function generateMesh(elements, bbox, altitude) {
-    // console.log(elements);
-
+    var roofGeometry = new THREE.Geometry(); // for the roof
```

Comment:
What's the difference with `var geometry = new THREE.Geometry(); // for the roof ` 2 lines below?